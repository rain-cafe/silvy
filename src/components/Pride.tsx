import styles from './Pride.module.scss';

export function Pride() {
    return (
        <div className={styles.pride} />
    );
}