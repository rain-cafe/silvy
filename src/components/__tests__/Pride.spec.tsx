import { Pride } from '../Pride'
import { createRenderComponent } from '../../../__tests__/render'

describe('Component(Pride)', () => {
    const renderPride = createRenderComponent(Pride);

    it('should display the Pride', () => {
        const { baseElement } = renderPride()

        expect(baseElement).toMatchSnapshot()
    });
});
