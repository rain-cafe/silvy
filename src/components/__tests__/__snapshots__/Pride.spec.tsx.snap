// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component(Pride) should display the Pride 1`] = `
<body>
  <div>
    <div
      class="pride"
    />
  </div>
</body>
`;
